packages/muze-utils/src/DataSystem/selection.js

```diff
@@ -34,7 +41,7 @@ class Selection {
         const updatedData = this._updatedata;
 
         newData.forEach((...params) => {
-            const key = this.idGetter ? this._idGetter(...params) : params[1];
+            const key = this._idGetter ? this._idGetter(...params) : params[1];
```

Comment:
It would make sense to store this._idGetter in a variable since it's being referenced multiple times in the loop